doc/material/v1.1.0.rst

```diff
@@ -0,0 +1,78 @@
+Version 1.1.0 (August 27, 2024)
+-------------------------------
+
+This is a release of MESSAGEix-Materials and includes model additions and enhancements.
+There are also a few of changes in the command line interface.
+
+Model changes
+~~~~~~~~~~~~~
+**Iron & steel sector**
```

Comment:
```suggestion
~~~~~~~~~~~~~

**Iron & steel sector**
```
For a heading to work, it might require the next line to be empty, but I'm not too sure on that one.